Packages/Tracking Preview/PhysicsHands/Runtime/PhysicsBone.cs

```diff
@@ -400,44 +443,100 @@ private bool RemoveGrabQueue(Rigidbody rigid)
             return true;
         }
 
-        private bool IsObjectGrabbable(HashSet<Collider> colliders)
+
+
+        private void UpdateBoneHoverDistances()
         {
-            foreach (var collider in colliders)
+            _objectDistance = float.MaxValue;
+
+            float distance;
+            Vector3 colliderPos, bonePos, midPoint, direction;
+            foreach (var hoverPairs in _hoverObjects)
             {
-                // Center and the base of the bone
-                if (Vector3.Dot(_grabDirectionCenter, (collider.ClosestPoint(_grabPositionCenter) - _grabPositionCenter).normalized) > 0.18f || Vector3.Dot(_grabDirection, (collider.ClosestPoint(_grabPositionBase) - _grabPositionBase).normalized) > 0.2f)
+                foreach (var hoveredCollider in hoverPairs.Value)
                 {
-                    return true;
+                    colliderPos = hoveredCollider.ClosestPoint(_collider.transform.position);
+                    bonePos = _collider.ClosestPoint(hoveredCollider.transform.position);
+
+                    midPoint = colliderPos + (bonePos - colliderPos) / 2f;
+
+                    colliderPos = hoveredCollider.ClosestPoint(midPoint);
+                    bonePos = _collider.ClosestPoint(midPoint);
+
+
+                    distance = Vector3.Distance(colliderPos, bonePos);
+                    direction = (colliderPos - bonePos).normalized;
+
+                    if (_hoverDirections[hoverPairs.Key].ContainsKey(hoveredCollider))
+                    {
+                        _hoverDirections[hoverPairs.Key][hoveredCollider].direction = direction;
+                        _hoverDirections[hoverPairs.Key][hoveredCollider].bonePos = bonePos;
+                        _hoverDirections[hoverPairs.Key][hoveredCollider].colliderPos = colliderPos;
+                    }
```

Comment:
colliderPos is removed now
```suggestion
                    {
                        _hoverDirections[hoverPairs.Key][hoveredCollider].direction = direction;
                        _hoverDirections[hoverPairs.Key][hoveredCollider].bonePos = bonePos;
                    }
```

Reply:
d'oh sorry - thought my project had recompiled but it hadn't :(